From c4459bf064c35019daed760459cace83ca559653 Mon Sep 17 00:00:00 2001
From: "sky.zhou" <sky.zhou@amlogic.com>
Date: Wed, 26 Feb 2014 15:31:12 +0800
Subject: [PATCH 3544/5965] PD#87358: add tripe buffer for g33 ,and set
 TARGET_USE_TRIPLE_FB_BUFFERS := true in device/amlogic.git

---
 arch/arm/boot/dts/amlogic/meson6_g33_1G.dtd   | 4 ++--
 arch/arm/boot/dts/amlogic/meson6_g33_512M.dtd | 4 ++--
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/arch/arm/boot/dts/amlogic/meson6_g33_1G.dtd b/arch/arm/boot/dts/amlogic/meson6_g33_1G.dtd
index aabab7ef9ab2..fb94c40455e6 100755
--- a/arch/arm/boot/dts/amlogic/meson6_g33_1G.dtd
+++ b/arch/arm/boot/dts/amlogic/meson6_g33_1G.dtd
@@ -111,10 +111,10 @@ void root_func(){
 		compatible = "amlogic,mesonfb";
 		dev_name = "mesonfb";
 		status = "okay";
-		reserve-memory = <0x00500000 0x00100000>;
+        reserve-memory = <0x0900000  0x00100000>;//8M+1M=9M
 		reserve-iomap = "true";
  		vmode = <1>; /**0:VMODE_720P 1:VMODE_LCD  2:VMODE_LVDS_1080P*/
- 		display_size_default = <1024 600 1024 1200 32>;
+ 		display_size_default = <1024 600 1024 1200 32>;//1024x768x4x3==0x708000,8MB
 	};
 	
 //$$ DEVICE = "mesonfb-ext"
diff --git a/arch/arm/boot/dts/amlogic/meson6_g33_512M.dtd b/arch/arm/boot/dts/amlogic/meson6_g33_512M.dtd
index 96291408c97f..50dfd7103880 100755
--- a/arch/arm/boot/dts/amlogic/meson6_g33_512M.dtd
+++ b/arch/arm/boot/dts/amlogic/meson6_g33_512M.dtd
@@ -111,10 +111,10 @@ void root_func(){
 		compatible = "amlogic,mesonfb";
 		dev_name = "mesonfb";
 		status = "okay";
-		reserve-memory = <0x00500000 0x00100000>;
+        reserve-memory = <0x0900000  0x00100000>;//8M+1M=9M
 		reserve-iomap = "true";
  		vmode = <1>; /**0:VMODE_720P 1:VMODE_LCD  2:VMODE_LVDS_1080P*/
- 		display_size_default = <1024 600 1024 1200 32>;//1024x768x4x2==0x04b0000,4.68MB
+ 		display_size_default = <1024 600 1024 1200 32>;//1024x768x4x3==0x708000,8MB
 	};
 	
 //$$ DEVICE = "mesonfb-ext"
-- 
2.19.0

